command/v7/service_command.go

```diff
@@ -32,7 +33,12 @@ func (cmd ServiceCommand) Execute(args []string) error {
 		err      error
 	)
 
-	cmd.serviceInstance, warnings, err = cmd.Actor.GetServiceInstanceDetails(string(cmd.RequiredArgs.ServiceInstance), cmd.Config.TargetedSpace().GUID, false)
+	switch {
```

Comment:
I don't like having two `switch` statements here - it feels like overlapping code paths. I prefer:
```go
switch {
case cmd.ShowGUID:
  return cmd.fetchAndDisplayGUID()
case cmd.Params:
  return cmd.fetchAndDisplayParams()
default:
  return cmd.fetchAndDisplayDetails()
}
```
So the code path split is explicit.